src/fragments/lib/graphqlapi/js/setup/connect.mdx

```diff
@@ -172,21 +166,25 @@ Amplify.configure({
 });
 ```
 
-### Recap
-At this point, your client is now configured to point to your AppSync API endpoint and you updated your authentication details based on the auth types you are using.
+</Accordion>
+
+<br/>
 
-[comment]: # (ADD TO RECAP Tie-in to real-world use and application of the topic discussed OR reiterate the benefit.)
+**Recap:** At this point, your client is now configured to point to your AWS AppSync API endpoint and you updated your authentication details based on the auth types you are using.
 
 ## Connect your application code to the data backend
 
-In this job, you will connect your application code to the data stack by connecting and configuring the Amplify Libraries. We will review how to install and configure Amplify Libraries to help you complete this task.
-> **Before you begin, you will need:**
-> - npm installed. For guidance on setting up npm the first time, see "([Getting started](https://docs.npmjs.com/getting-started) on the npm Docs site.)
-> - A data stack deployed
+You will connect your application code to the data stack by connecting and configuring the Amplify Libraries. We will review how to install and configure Amplify Libraries to help you complete this task. Before starting these steps you should [review the required configuration](#Review-the-required-configuration).
+
```

Comment:
This section below reads out of place and needs a better tie in to the intro above and what follows. From the intro I was expecting to dive right into connecting and configuring libraries not using codegen. Can we update this section to work together more so it reads less out of place? Also, is this only for JS/TS or do we have it for all frameworks? Just wondering if this section will have different content for different frameworks?

Reply:
I agree this reads out of place and I think the placement of this might change as I wrap up the other sections. I'd recommend adding a "TODO" to come back later to see the best place to put this.